File: CursoPOOUber/Java/UberBlack.java
package CursoPOOUber.Java;
import java.util.ArrayList;
import java.util.Map;
class UberBlack extends Car {
    Map<String, Map<String,Integer>>typeCarAccepted;
    ArrayList<String> seatsMaterial;
    public UberBlack(String license, Account driver, Map<String, 
    Map<String,Integer>>typeCarAccepted, ArrayList<String> seatsMaterial ){
        super(license,driver); // Atributos de la super clase en este caso seria car seria 
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }
    
}

File: CursoPOOUber/JS/UberBlack.js
class UberBlack extends car {
    constructor(license, driver, typeCarAccepted, seatsMaterial){ // metodo de contructor
        super(license, driver) //herencia de la clase car.js
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}